package.json

```diff
@@ -47,72 +47,72 @@
     "react-codemirror2": "5.1.0",
     "react-infinite-scroll-component": "4.2.0",
     "react-tether": "1.0.4",
-    "react-textarea-autosize": "7.0.4",
-    "unidiff": "0.0.4"
+    "react-textarea-autosize": "7.1.0",
+    "unidiff": "1.0.1"

```

Comment:
im not sure if this one can create problems

Reply:
Hard to tell... the packages doesn't have any release notes
![image](https://user-images.githubusercontent.com/35579930/51055526-1148fe80-15ae-11e9-9874-3ab7303301c3.png)